include/bluetooth/mesh/access.h

```diff
@@ -1,5 +1,5 @@
 /** @file
- *  @brief Bluetooth Mesh Access Layer APIs.
+ *  @brief Access layer APIs.
```

Comment:
Just a question... is there any impact to us removing Bluetooth mesh from these files?

Reply:
This is just doxygen file name, they should have the name of the file. It does not matter much, maybe the best had been to remove it. But I would rather leave it as this.